woocommerce-ac.php

```diff
@@ -1956,12 +1989,20 @@ public function wcal_email_track_links( $template ) {
 
 				if ( $user_id < '63000000' ) {
 					$user_login = $user->data->user_login;
-					wp_set_auth_cookie( $user_id );
-					$my_temp = wc_load_persistent_cart( $user_login, $user );
-					do_action( 'wp_login', $user_login, $user );
-					if ( isset( $sign_in ) && is_wp_error( $sign_in ) ) {
-						echo esc_html( $sign_in->get_error_message() );
-						exit;
+					if ( 'on' === get_option( 'wcal_auto_login_users', 'on' ) ) {
+						wp_set_auth_cookie( $user_id );
+						$my_temp = wc_load_persistent_cart( $user_login, $user );
+						do_action( 'wp_login', $user_login, $user );
+						if ( isset( $sign_in ) && is_wp_error( $sign_in ) ) {
+							echo esc_html( $sign_in->get_error_message() );
+							exit;
+						}
+					} else {
+						wp_set_auth_cookie( $user_id, false, '', 'loggedout' );
+						$my_temp = wc_load_persistent_cart( $user_login, $user );
+						set_transient( 'wcal_email_sent_id', $email_sent_id, 1800 );
```

Comment:
The feature seems to be incompletely coded. The transients set here are not used anywhere @nidhitatosaniya 

Please let me know if I've missed anything.